libs/ui-libraries/src/lib/repo-info/repo-info.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="ml-5 justify-content-start h-100 col-2 flex-column d-flex test">
+    <div class="profile w-100 flex-column border-bottom">
+        <div class="pic"></div>
+        <h3 style="color: #5B5B5B" >John Doe</h3>
+        <h3 style="color: #9B9C9B">jDizzle</h3>
+    </div>
+
+    <div class="user-info d-flex flex-column w-100 border-bottom">
+        <h3>Peoria, AZ</h3>
```

Comment:
Let's mock out as much of this data as possible in the component if the services aren't ready.